### PR TITLE
REL: Version bump URLS (pt 1 of 2)

### PR DESCRIPTION
**Pull request by thermokarst**

### CODE REVIEW

**Comment by jairideout** [comment]
I'll get you the new spreadsheet URLs tomorrow 👍

File: urls.py [hunk 1]
```diff
@@ -36,7 +40,18 @@
...
     'distro/core/qiime20174-1492814818.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20174-1492814818.zip',
-    
+
+    # 2017.5 DISTRO
+    'distro/core/qiime2-2017.5-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.5/release/qiime2-2017.5-conda-osx-64.txt',
+    'distro/core/qiime2-2017.5-conda-linux-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.5/release/qiime2-2017.5-conda-linux-64.txt',
+    ### TODO: Update these URLs
```
Comment (thermokarst) [inline urls.py:49]: Actually, do we *need* these to be data links? We could just put the S3 links into distro/core/virtualbox-images.txt directly, then it wouldn't require a 2-part PR for version bumps.

Comment (thermokarst) [reply urls.py:49]: Discussed with @jairideout & @ebolyen: we will continue to provide VBox downloads as data.qiime2.org links for now, but we will probably revisit in the future!

**Comment by jairideout** [comment]
New spreadsheet keys:

- Moving Pictures: `1_cDVn4B1wjzxdirkSxoEvOEmWPGn0XyCqbQGpeyg4Uc`
- FMT: `1kv1pomvbr3WQiEUA8P2N6zyVHkue4XXRI_U_3UmJVvw`
- Atacama: `1GRaNc-Ik7_4K45EQjCjrML7dadxoz6l-w8CF4I4T7Ug`

**Review by jairideout — APPROVED** [review]

**Pull request merged**